### PR TITLE
model updates to support org IDs, ad IDs, asnet6, and various additional fields

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/infotech.py [hunk 1]
```diff
@@ -191,10 +191,13 @@ def getModelDefs(self):
                     'doc': 'A arbitrary, unversioned software product.',
                 }),
 
-                ('it:os:ios:idfa', ('str', {'lower': 1}), {
+                ('it:adid', ('str', {'lower': 1}), {
```
Comment (vEpiphyte) [inline synapse/models/infotech.py:194]: strip: true.

File: synapse/models/orgs.py [hunk 2]
```diff
@@ -76,8 +76,56 @@ def getModelDefs(self):
                 ('ou:campaign', ('guid', {}), {
                     'doc': 'Represents an orgs activity in pursuit of a goal.',
                 }),
+                ('ou:id:type', ('guid', {}), {
+                    'doc': 'A type of id number issued by an org.',
+                }),
+                ('ou:id:value', ('str', {'strip': True}), {
```
Comment (vEpiphyte) [inline synapse/models/orgs.py:82]: do we care about case here? ``DL1234`` vs ``dl1324``

**Review by whippit — APPROVED** [review]

File: synapse/tests/test_model_telco.py [hunk 3]
```diff
@@ -94,8 +94,10 @@ async def test_telco_simple(self):
                          'mac': '00:00:00:00:00:00',
                          'ipv4': '1.2.3.4',
                          'ipv6': '::1',
-                         'wifi:ssid': 'The Best SSID2',
-                         'wifi:bssid': '00:11:22:33:44:55',
+                         'wifi': ('The Best SSID2', '00:11:22:33:44:55'),
+                         #'wifi:ssid': 'The Best SSID2',
```
Comment (vEpiphyte) [inline synapse/tests/test_model_telco.py:98]: remove dead code

**Review by vEpiphyte — CHANGES_REQUESTED** [review]
see comments

File: synapse/models/orgs.py (hunk 2 above)
Comment (invisig0th) [reply synapse/models/orgs.py:82]: Unfortunately, we can't predict whether an org will create an ID that is case sensitive.  Lets discuss.

Comment (invisig0th) [reply synapse/models/orgs.py:82]: Gonna leave it case sensitive.  Too many potential issues with normalization. :D

**Review by Cisphyx — APPROVED** [review]

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**